### PR TITLE
#1300 Add right alignment function in sql email content

### PR DESCRIPTION
**Pull request by Yeleights**
## What is the purpose of the pull request
*#1300 Add right alignment function in sql email content*
## Brief change log
  - *Modify alert.utils.ExcelUtils*
  - *Modify alert.resources.mail_templates.alert_mail_template.ftl*
## Verify this pull request
This pull request is already covered by existing MailUtilsTest.

### CODE REVIEW

**Comment by Yeleights** [comment]
@qiaozhanwei @Technoboy-

File: dolphinscheduler-alert/src/main/resources/mail_templates/alert_mail_template.ftl [hunk 1]
```diff
@@ -14,4 +14,40 @@
...
\ No newline at end of file
+<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01 Transitional//EN' 'http://www.w3.org/TR/html4/loose.dtd'>
```
Comment (lgcareer) [inline dolphinscheduler-alert/src/main/resources/mail_templates/alert_mail_template.ftl]: After formatting, you can verify whether the content of the sent email has spaces and blank lines.

Comment (Yeleights) [reply dolphinscheduler-alert/src/main/resources/mail_templates/alert_mail_template.ftl]: thx, i found this problem. i will fix

Comment (Yeleights) [reply dolphinscheduler-alert/src/main/resources/mail_templates/alert_mail_template.ftl]: I cancel format file: alert.resources.mail_templates.alert_mail_template.ftl

Comment (lgcareer) [reply dolphinscheduler-alert/src/main/resources/mail_templates/alert_mail_template.ftl]: Ok.

**Pull request merged**